### PR TITLE
Adding LogAnalytics to Compute spec

### PR DESCRIPTION
**Pull request by waynekuo**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by huangpf** [comment]
@AutorestCI 
Can you do a commit to Azure/azure-sdk-for-net?

**Review by jianghaolu — CHANGES_REQUESTED** [review]
You need to reference the examples from the Swagger spec using `x-ms-examples`.

**Comment by waynekuo** [comment]
Added references to example files.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsThrottledRequests.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          "toTime": "2018-01-23T01:54:06.862601Z",
+          "groupByOperationName": true,
+          "groupByResourceName": false
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
```
Comment (jianghaolu) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsThrottledRequests.json]: In the Swagger, 200 doesn't return a response body.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          "blobContainerSasUri": "https://somesasuri",
+          "fromTime": "2018-01-21T01:54:06.862601Z",
+          "toTime": "2018-01-23T01:54:06.862601Z",
+          "groupByResourceName": true
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
```
Comment (jianghaolu) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json]: In the Swagger, 200 doesn't return a response body.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsThrottledRequests.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "api-version": "2017-12-01",
+    "parameters": {
+      "location": "westus",
```
Comment (jianghaolu) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsThrottledRequests.json]: "location" is a top-level parameter.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "api-version": "2017-12-01",
+    "parameters": {
+      "location": "westus",
```
Comment (jianghaolu) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json]: "location" is a top-level parameter.

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "location": "westus",
+    "api-version": "2017-12-01",
+    "parameters": {
+      "properties": {
```
Comment (jianghaolu) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json]: Please extract everything out of both "RequestRateByIntervalInput" and "properties". The "parameters" section should contain the parameters in this format:
```json
"<parameter-name>": <actual payload>
```

In this case, the actual payload is 
```json
{
  "intervalLength": "FiveMins",
  "blobContainerSasUri": "https://somesasuri",
  "fromTime": "2018-01-21T01:54:06.862601Z",
  "toTime": "2018-01-23T01:54:06.862601Z",
  "groupByResourceName": true
 }
```

You can see another example in compute here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/CreateAScaleSetFromACustomImage.json#L8

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": ""
+    },
+    "202": {
+      "body": {
```
Comment (jianghaolu) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json]: 202 doesn't have a body either :) Same for the other example.

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json (hunk 5 above)
Comment (jianghaolu) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json]: Same for the other example.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsThrottledRequests.json (hunk 1 above)
Comment (waynekuo) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsThrottledRequests.json]: Updated the Swagger file now. 200 should return a response body. Missed the copy from our internal repo.

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**